tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/jvmmodel/TclParameterGeneratorIntegrationTest.xtend

```diff
@@ -67,6 +78,17 @@ class TclParameterGeneratorIntegrationTest extends AbstractTclGeneratorIntegrati
 			public class MyTest extends AbstractTestCase {
 			  private DummyFixture dummyFixture = new DummyFixture();
 			  
+			  private MaskingString env_confEnvVar = new MaskingString(System.getenv("confEnvVar"));
+			  
+			  private String env_nonConfEnvVar = System.getenv("nonConfEnvVar");
+			  
+			  @Before
+			  public void checkEnvironmentVariablesOnExistence() throws Exception {
+			    org.junit.Assert.assertNotNull("environment variable 'confEnvVar' must not be null", env_confEnvVar.get());
+			    org.junit.Assert.assertNotNull("environment variable 'nonConfEnvVar' must not be null", env_nonConfEnvVar);
```

Comment:
Why not import org.junit.Assert.... statically?

Reply:
I get it, its generated Java code, and the generator produces fully-qualified names so it does not have to be "smart" about import statements.